diff -ur a/tests/test_x509_ext.py b/tests/test_x509_ext.py
--- a/tests/test_x509_ext.py
+++ b/tests/test_x509_ext.py
@@ -3003,7 +3003,8 @@
         with pytest.raises(ValueError):
             x509.DistributionPoint("data", "notname", None, None)
 
-    def test_relative_name_name_value_deprecated(self):
+    def DISABLED_test_relative_name_name_value_deprecated(self):
+        # pytest 2.7 does not support context manager for deprecated_call()
         with pytest.deprecated_call():
             x509.DistributionPoint(
                 None,
diff -ur a/tests/test_x509.py b/tests/test_x509.py
--- a/tests/test_x509.py
+++ b/tests/test_x509.py
@@ -527,7 +527,8 @@
             assert cert.serial == 2
             assert cert.serial_number == 2
 
-    def test_cert_serial_warning(self, backend):
+    def DISABLED_test_cert_serial_warning(self, backend):
+        # pytest 2.7 does not support context manager for deprecated_call()
         cert = _load_cert(
             os.path.join("x509", "PKITS_data", "certs", "GoodCACert.crt"),
             x509.load_der_x509_certificate,
