diff --git a/setup.py b/setup.py
--- a/setup.py
+++ b/setup.py
@@ -37,7 +37,7 @@ requirements = [
     "idna>=2.0",
     "pyasn1>=0.1.8",
     "six>=1.4.1",
-    "setuptools>=11.3",
+    "setuptools",
 ]
 setup_requirements = []

@@ -58,7 +58,7 @@
     setup_requirements.append("cffi>=1.4.1")
 
 test_requirements = [
-    "pytest>=2.9.0",
+    "pytest",
     "pretend",
     "iso8601",
     "pyasn1_modules",
